package.json

```diff
@@ -34,7 +34,7 @@
     "@carbon/icons-react": "~10.11.0",
     "@carbon/themes": "~10.12.0",
     "@data-driven-forms/pf3-component-mapper": "~2.8.13",
-    "@data-driven-forms/react-form-renderer": "~2.8.13",
+    "@data-driven-forms/react-form-renderer": "~2.8.21",
```

Comment:
If you really need an newer version, you should also update the `pf3-component-mapper` as their versions should always match. But I guess you don't need that change here, because you're not introducing any new components.